### ISSUE TITLE
[ZH] One more position where Yasser tricked SF

### ISSUE THREAD

**Issue by Vinvin20** [ddugovic/Stockfish#543]
23.Rxf8! and black seems lost here : https://lichess.org/KVOOLOgC#45
But SF evaluates this position as -+ for a long time.
r2q1R2/pppk2Bp/2n1p3/3p3b/7b/2NPpPP1/PP2Qp1P/R4K1R/NBppn b - - 45 23

**Comment by ddugovic** [comment]
Results from latest `master` (bearing in mind that evaluations only exist for debugging; it's the variations that matter):

```
info depth 1 seldepth 1 multipv 1 score cp 921 nodes 170 nps 85000 tbhits 0 time 2 pv d8f8 g7f8 a8f8 e2e3
info depth 2 seldepth 2 multipv 1 score cp 921 nodes 301 nps 150500 tbhits 0 time 2 pv d8f8 g7f8
info depth 3 seldepth 4 multipv 1 score cp 516 nodes 740 nps 246666 tbhits 0 time 3 pv d8g5 f8a8 N@d2 e2d2 e3d2
info depth 4 seldepth 5 multipv 1 score cp 355 nodes 1350 nps 337500 tbhits 0 time 4 pv d8f8 g7f8 d5d4 Q@g7 d7c8 g7h7
info depth 5 seldepth 6 multipv 1 score cp 630 nodes 2906 nps 581200 tbhits 0 time 5 pv d8f8 g7f8 h5f3 e2e3 P@e2 f1f2 a8f8
info depth 6 seldepth 8 multipv 1 score cp -88 nodes 11888 nps 660444 tbhits 0 time 18 pv d5d4 N@c5 d7c8 f1g2 N@e8 c5e6 d8d7
info depth 7 seldepth 10 multipv 1 score cp -24 nodes 18718 nps 668500 tbhits 0 time 28 pv d8f8 g7f8 R@e1 a1e1 f2e1q e2e1 a8f8 N@c5 d7c8 R@a8 N@b8 g3h4 f8f3 f1g2
info depth 8 seldepth 12 multipv 1 score cp 1328 nodes 23895 nps 663750 tbhits 0 time 36 pv d8f8 e2e3 f8g7 g3h4 a8f8 N@c5 d7c8 P@d7 g7d7 c5d7 c8d7 f1f2
info depth 9 seldepth 15 multipv 1 score cp 1906 nodes 36804 nps 657214 tbhits 0 time 56 pv d8f8 e2e3 R@e1 a1e1 f2e1q f1e1 P@f2 e3f2 P@d2 e1d2 h4g5 d2d1 h5f3 f2f3 f8f3 d1e1
info depth 10 seldepth 18 multipv 1 score cp 1553 nodes 66468 nps 658099 tbhits 0 time 101 pv d8f8 e2e3 f8g7 B@h3 d7c8 f1g2 P@d7 P@g4 d5d4 e3f2 N@e3 f2e3
info depth 11 seldepth 22 multipv 1 score cp 1439 nodes 170597 nps 638940 tbhits 0 time 267 pv d8f8 e2e3 f8g7 B@h3 N@d4 c3d5 d7c8 h3e6 d4e6 P@g2 P@d7 g3h4 g7b2
info depth 12 seldepth 26 multipv 1 score cp 791 nodes 468924 nps 632825 tbhits 0 time 741 pv d8f8 g7f8 R@g1 h1g1 f2g1q f1g1 P@f2 g1h1 R@g1 a1g1 f2g1q h1g1 P@f2 g1g2 h5f3 g2f3 a8f8 f3e3
info depth 13 seldepth 33 multipv 1 score cp 1487 nodes 711148 nps 640673 hashfull 334 tbhits 0 time 1110 pv d8f8 g7f8 R@g1 h1g1 f2g1q f1g1 P@f2 g1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g2 P@h3 g2h1 R@g1 a1g1 f2g1q h1g1 g5f3 g1h1 a8f8
info depth 14 seldepth 33 multipv 1 score cp 320 nodes 1169588 nps 648689 hashfull 516 tbhits 0 time 1803 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 c6e5 g4h5 g5f3 B@a4 P@c6 a4c6 e5c6 Q@g7 B@f7 g7f7 P@e7 h1e1
info depth 15 seldepth 29 multipv 1 score cp 582 nodes 1592792 nps 645639 hashfull 658 tbhits 0 time 2467 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 c6e5 g4h5 g5f3 B@a4 P@b5 a4b5 P@c6 b5c6 b7c6 N@c5 d7e8 P@f7 e8f8 B@h6 f8e7 f7f8q a8f8 h6f8 e7f8 c5e6 f8e7 g3h4
info depth 16 seldepth 34 multipv 1 score cp 591 lowerbound nodes 1952547 nps 642285 hashfull 763 tbhits 0 time 3040 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 c6e5 g4h5 g5f3 B@a4 P@c6 a4c6 b7c6 N@c5 d7e8 P@d7
info depth 16 seldepth 34 multipv 1 score cp 600 lowerbound nodes 1985608 nps 641968 hashfull 769 tbhits 0 time 3093 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 c6e5 g4h5 g5f3 P@a6
info depth 16 seldepth 34 multipv 1 score cp 613 lowerbound nodes 1994679 nps 642201 hashfull 771 tbhits 0 time 3106 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 c6e5 g4h5 g5f3 P@a6
info depth 16 seldepth 34 multipv 1 score cp 631 lowerbound nodes 2011679 nps 642093 hashfull 773 tbhits 0 time 3133 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 c6e5 g4h5 g5f3 Q@g7
info depth 16 seldepth 34 multipv 1 score cp 656 lowerbound nodes 2026598 nps 641937 hashfull 778 tbhits 0 time 3157 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 c6e5 g4h5 g5f3 B@a4 P@c6 a4c6 b7c6 N@c5 d7e8 P@f7
info depth 16 seldepth 34 multipv 1 score cp 690 lowerbound nodes 2043732 nps 642279 hashfull 780 tbhits 0 time 3182 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 c6e5 g4h5 g5f3 B@a4 P@c6 a4c6 b7c6 N@c5 d7e8 P@f7
info depth 16 seldepth 34 multipv 1 score cp 734 lowerbound nodes 2065691 nps 642516 hashfull 786 tbhits 0 time 3215 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 c6e5 g4h5 g5f3 P@f5
info depth 16 seldepth 34 multipv 1 score cp 792 lowerbound nodes 2094730 nps 642555 hashfull 788 tbhits 0 time 3260 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 c6e5 g4h5 g5f3 B@a4 P@c6 a4c6 b7c6 N@c5 d7e8 P@d7
info depth 16 seldepth 34 multipv 1 score cp 867 lowerbound nodes 2358151 nps 642197 hashfull 839 tbhits 0 time 3672 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 c6e5 g4h5 g5f3 B@a4 P@c6 a4c6 b7c6 N@c5 d7e8 P@f7
info depth 16 seldepth 34 multipv 1 score cp 963 lowerbound nodes 2463859 nps 640962 hashfull 862 tbhits 0 time 3844 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 c6e5 g4h5 g5f3 B@a4 P@c6 a4c6 b7c6 N@c5 d7e8 P@d7
info depth 16 seldepth 34 multipv 1 score cp 899 nodes 2530803 nps 641521 hashfull 868 tbhits 0 time 3945 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 c6e5 g4h5 g5f3 B@a4 P@c6 a4c6 b7c6 N@c5 d7e8 P@d7 e8f8 c5e6 f8e7 Q@e8 a8e8 d7e8q e7e8 e6c7 e8f7 P@e6 f7g8 R@e8 B@f8 P@f7 e5f7 e6f7 g8f7 g3h4
info depth 17 seldepth 40 multipv 1 score cp 907 lowerbound nodes 3778501 nps 630380 hashfull 957 tbhits 0 time 5994 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 c6e5 g4h5 g5f3 B@a4 P@c6 a4c6 b7c6 N@c5 d7e8 P@d7 e8f8 c5e6 f8e7 Q@e8 a8e8 d7e8q e7e8 e6c7 e8f7 P@e6 f7e7 P@f6
info depth 17 seldepth 40 multipv 1 score cp 889 upperbound nodes 3974300 nps 631141 hashfull 962 tbhits 0 time 6297 pv d8f8 g7f8
info depth 17 seldepth 40 multipv 1 score cp 877 upperbound nodes 4324840 nps 633212 hashfull 974 tbhits 0 time 6830 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 c6e5 g4h5 P@c2
info depth 17 seldepth 40 multipv 1 score cp 858 upperbound nodes 4392531 nps 633751 hashfull 975 tbhits 0 time 6931 pv d8f8 g7f8 P@g2
info depth 17 seldepth 40 multipv 1 score cp 833 upperbound nodes 4451838 nps 634345 hashfull 976 tbhits 0 time 7018 pv d8f8 g7f8 R@e1 f1g2 f2f1q
info depth 17 seldepth 40 multipv 1 score cp 799 upperbound nodes 4536535 nps 634657 hashfull 979 tbhits 0 time 7148 pv d8f8 g7f8
info depth 17 seldepth 40 multipv 1 score cp 754 upperbound nodes 4592543 nps 635118 hashfull 979 tbhits 0 time 7231 pv d8f8 g7f8
info depth 17 seldepth 40 multipv 1 score cp 696 upperbound nodes 4685557 nps 636451 hashfull 982 tbhits 0 time 7362 pv d8f8 g7f8
info depth 17 seldepth 40 multipv 1 score cp 622 upperbound nodes 4780067 nps 637172 hashfull 982 tbhits 0 time 7502 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 e1g1
info depth 17 seldepth 40 multipv 1 score cp 526 upperbound nodes 4902747 nps 636720 hashfull 983 tbhits 0 time 7700 pv d8f8 g7f8
info depth 17 seldepth 40 multipv 1 score cp 404 upperbound nodes 5690601 nps 635892 hashfull 998 tbhits 0 time 8949 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g1 h3g4 P@f5
info depth 17 seldepth 40 multipv 1 score cp 460 nodes 5811711 nps 636411 hashfull 998 tbhits 0 time 9132 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g1 h3g4 c6e5 g4h5 g1f3 B@a4 P@c6 a4c6 b7c6 N@c5 d7e8 P@d7 e5d7 P@f7 e8f8 c5d7 f8e7 f7f8q a8f8 B@a3 e7d7 N@c5 d7c8 Q@b7 c8d8 c5e6 d8d7 e6f8 d7d8 g3h4
info depth 18 seldepth 38 multipv 1 score cp 469 lowerbound nodes 7953473 nps 631127 hashfull 999 tbhits 0 time 12602 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g1 h3g4 c6e5 g4h5 g1f3 B@a4 P@c6 a4c6 b7c6 N@c5 d7e8 P@d7 e5d7 P@f7 e8f8 c5d7 f8e7 f7f8q a8f8 a1e1
info depth 18 seldepth 38 multipv 1 score cp 478 lowerbound nodes 7987845 nps 631100 hashfull 999 tbhits 0 time 12657 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g1 h3g4 c6e5 g4h5 g1f3 B@a4 P@c6 a4c6 b7c6 N@c5 d7e8 P@d7 e5d7 P@f7 e8f8 c5e6
info depth 18 seldepth 38 multipv 1 score cp 491 lowerbound nodes 7999378 nps 631164 hashfull 999 tbhits 0 time 12674 pv d8f8 g7f8 R@e1 a1e1
info depth 18 seldepth 38 multipv 1 score cp 509 lowerbound nodes 8006954 nps 631264 hashfull 999 tbhits 0 time 12684 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g1 h3g4 c6e5 g4h5 g1f3 Q@g7
info depth 18 seldepth 38 multipv 1 score cp 535 lowerbound nodes 8012442 nps 631348 hashfull 999 tbhits 0 time 12691 pv d8f8
info depth 18 seldepth 38 multipv 1 score cp 570 lowerbound nodes 8017949 nps 631483 hashfull 999 tbhits 0 time 12697 pv d8f8
info depth 18 seldepth 38 multipv 1 score cp 614 lowerbound nodes 8033733 nps 631582 hashfull 999 tbhits 0 time 12720 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g1 h3g4 c6e5 g4h5 g1f3 B@a4 P@c6 N@d8
info depth 18 seldepth 38 multipv 1 score cp 672 lowerbound nodes 8084960 nps 631637 hashfull 999 tbhits 0 time 12800 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g1 h3g4 c6e5 g4h5 g1f3 N@c5
info depth 18 seldepth 38 multipv 1 score cp 747 lowerbound nodes 8104274 nps 631813 hashfull 999 tbhits 0 time 12827 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g1 h3g4 c6e5 g4h5 g1f3 N@c5
info depth 18 seldepth 38 multipv 1 score cp 842 lowerbound nodes 8223403 nps 631986 hashfull 999 tbhits 0 time 13012 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g1 h3g4 c6e5 g4h5 g1f3 Q@g7 P@e7 B@b5
info depth 18 seldepth 39 multipv 1 score cp 749 nodes 8493398 nps 632372 hashfull 999 tbhits 0 time 13431 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g1 h3g4 c6e5 g4h5 g1f3 B@b5 P@c6 b5c6 b7c6 N@c5 d7e8 P@d7 e5d7 P@f7 e8f8 c5e6 f8f7 Q@g7 f7e6 B@h3 e6d6 g7d7 d6c5 N@e6 c5b4 c3d5 c6d5 P@c3 b4a5 d7d5 a5a6 d5f3
info depth 19 seldepth 41 multipv 1 score cp 741 upperbound nodes 11679387 nps 637729 hashfull 999 tbhits 0 time 18314 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g1 h3g4 P@f5
info depth 19 seldepth 41 multipv 1 score cp 732 upperbound nodes 11910089 nps 637994 hashfull 999 tbhits 0 time 18668 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g1 h3g4 P@f5
info depth 19 seldepth 41 multipv 1 score cp 719 upperbound nodes 12055378 nps 638458 hashfull 999 tbhits 0 time 18882 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5
info depth 19 seldepth 41 multipv 1 score cp 700 upperbound nodes 12158342 nps 638971 hashfull 999 tbhits 0 time 19028 pv d8f8 g7f8 R@e1 f1g2 f2f1q
info depth 19 seldepth 41 multipv 1 score cp 675 upperbound nodes 12316924 nps 639441 hashfull 999 tbhits 0 time 19262 pv d8f8 g7f8 R@e1 f1g2 f2f1q
info depth 19 seldepth 41 multipv 1 score cp 642 upperbound nodes 12956756 nps 639681 hashfull 999 tbhits 0 time 20255 pv d8f8 g7f8 R@e1 f1g2 f2f1q
info depth 19 seldepth 41 multipv 1 score cp 597 upperbound nodes 13426459 nps 640514 hashfull 999 tbhits 0 time 20962 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 h4g3
info depth 19 seldepth 41 multipv 1 score cp 539 upperbound nodes 13755103 nps 640964 hashfull 999 tbhits 0 time 21460 pv d8f8 g7f8 R@e1 f1g2 f2f1q
info depth 19 seldepth 41 multipv 1 score cp 464 upperbound nodes 14502939 nps 641921 hashfull 999 tbhits 0 time 22593 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5
info depth 19 seldepth 41 multipv 1 score cp 369 upperbound nodes 15136554 nps 642550 hashfull 999 tbhits 0 time 23557 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 e1h1
info depth 19 seldepth 41 multipv 1 score cp 247 upperbound nodes 19697962 nps 641209 hashfull 999 tbhits 0 time 30720 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g1 h3g4 c6e5 g4h5 g1f3 B@b5 Q@c6
info depth 19 seldepth 41 multipv 1 score cp 92 upperbound nodes 27200108 nps 640454 hashfull 999 tbhits 0 time 42470 pv d8f8 g7f8 R@e1 f1g2 h5f3 e2f3 P@h3 g2h3 N@g1 h3g4 c6e5 g4h5 P@g6
info depth 19 seldepth 41 multipv 1 score cp 199 nodes 34627461 nps 638058 hashfull 999 tbhits 0 time 54270 pv d8f8 g7f8 R@e1 f1g2 f2f1q e2f1 h5f3 f1f3 P@h3 g2h3 N@g5 h3g4 c6e5 g4h5 g5f3 B@a4 P@c6 a4c6 b7c6 N@c5 d7e8 P@d7 e5d7 P@f7 e8f8 c5d7 f8e7 Q@e8 a8e8 f7e8q e7e8 R@f8 e8d7 B@e8 d7d8 N@b7 d8c8 e8c6 Q@e8 f8e8 N@d8
info depth 20 seldepth 43 multipv 1 score cp 190 upperbound nodes 50442215 nps 637572 hashfull 999 tbhits 0 time 79116 pv d8f8 g7f8 R@e1 f1g2 f2f1q e2f1 h5f3 f1f3 P@h3 g2h3 N@g5 h3g4 c6e5 g4h5 g5f3 B@a4 P@c6 a4c6 b7c6 N@c5 d7e8 P@d7 e5d7 P@f7 e8f8 c5d7 f8f7
info depth 20 seldepth 43 multipv 1 score cp 181 upperbound nodes 52771721 nps 637871 hashfull 999 tbhits 0 time 82731 pv d8f8 g7f8 R@e1 f1g2 f2f1q e2f1 h5f3 f1f3 P@h3 g2h3 N@g5 h3g4 P@h5
info depth 20 seldepth 45 multipv 1 score cp 168 upperbound nodes 54780266 nps 638025 hashfull 999 tbhits 0 time 85859 pv d8f8 g7f8 R@e1 f1g2 f2f1q e2f1 h5f3 f1f3 P@h3 g2h3 N@g5 h3g4 c6e5 g4h5 P@g6
info depth 20 seldepth 45 multipv 1 score cp 150 upperbound nodes 56890553 nps 638337 hashfull 999 tbhits 0 time 89123 pv d8f8 g7f8 R@e1 f1g2 f2f1q e2f1 h5f3 f1f3 P@h3 g2h3 N@g5 h3g4 c6e5 g4h5 g5f3 B@a4 P@c6 a4c6 b7c6 N@c5 d7e8 P@d7 e8f8
info depth 20 seldepth 45 multipv 1 score cp 125 upperbound nodes 59228600 nps 638514 hashfull 999 tbhits 0 time 92760 pv d8f8 g7f8 R@e1 f1g2 f2f1q e2f1 h5f3 f1f3 P@h3 g2h3 N@g5 h3g4 P@g6
info depth 20 seldepth 45 multipv 1 score cp 91 upperbound nodes 62100791 nps 638338 hashfull 999 tbhits 0 time 97285 pv d8f8 g7f8 R@e1 f1g2 f2f1q e2f1 h5f3 f1f3 P@h3 g2h3 N@g5 h3g4 c6e5 g4h5 g5f3 B@a4 P@c6 a4c6 e5c6
info depth 20 seldepth 45 multipv 1 score cp 117 lowerbound nodes 64168741 nps 638291 hashfull 999 tbhits 0 time 100532 pv d8f8 g7f8 P@g2 f1g2 h5f3 g2f3
info depth 20 seldepth 45 multipv 1 score cp 175 lowerbound nodes 65604040 nps 638562 hashfull 999 tbhits 0 time 102737 pv d8f8 g7f8 P@g2 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 h7h5 g4h5 R@h7 P@h6
info depth 20 seldepth 45 multipv 1 score cp 250 lowerbound nodes 65741213 nps 638468 hashfull 999 tbhits 0 time 102967 pv d8f8 g7f8 P@g2 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 h7h5 g4h5 R@h7 B@h6 g5f3 N@c5 d7c8 P@d7 h7d7 Q@e8 Q@d8 e8e6 P@f7 e6c6
info depth 20 seldepth 45 multipv 1 score cp 346 lowerbound nodes 66002868 nps 638479 hashfull 999 tbhits 0 time 103375 pv d8f8 g7f8 P@g2 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 h7h5 g4h5 R@h7 B@h6 g5f3 N@c5 d7c8 P@d7 h7d7 Q@e8 Q@d8 e8d8
info depth 20 seldepth 45 multipv 1 score cp 46 upperbound nodes 68913644 nps 638893 hashfull 999 tbhits 0 time 107864 pv d8f8 g7f8 P@g2 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 h7h5 g4h5 R@h7 B@h6 g5f3 N@c5 d7c8 P@d7 h7d7 Q@e8 Q@d8 e8e6 P@e5
info depth 20 seldepth 45 multipv 1 score cp 257 lowerbound nodes 69200862 nps 638531 hashfull 999 tbhits 0 time 108375 pv d8f8 g7f8 P@g2 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 h7h5 g4h5 R@h7 B@h6 g5f3 N@c5 d7c8 c3d5
info depth 20 seldepth 45 multipv 1 score cp 194 nodes 69335855 nps 638557 hashfull 999 tbhits 0 time 108582 pv d8f8 g7f8 P@g2 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 h7h5 g4h5 R@h7 B@h6 g5f3 N@c5 d7c8 P@d7 h7d7 Q@e8 Q@d8 e8e6 P@f7 e6c6 P@g6 h5g4 f3e5 g4h3 e5c6 P@a6 a8b8 a6b7 b8b7 B@g4 P@f5 g4e2 b7b2 P@b7 b2b7 h6e3 d8f8 g3h4
info depth 21 seldepth 42 multipv 1 score cp 185 upperbound nodes 74959716 nps 638726 hashfull 999 tbhits 0 time 117358 pv d8f8 g7f8 P@g2 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 h7h5 g4h5 R@h7 B@h6 g5f3 N@c5 d7c8 P@d7 h7d7 Q@e8 Q@d8 e8e6 P@f7 e6c6 P@g6 h5g4 f3e5 g4h3 e5c6 B@a6 Q@g4
info depth 21 seldepth 42 multipv 1 score cp 177 upperbound nodes 76500911 nps 639014 hashfull 999 tbhits 0 time 119717 pv d8f8 g7f8 P@g2 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 c6e5 g4h5 P@g6
info depth 21 seldepth 42 multipv 1 score cp 164 upperbound nodes 77894855 nps 639000 hashfull 999 tbhits 0 time 121901 pv d8f8 g7f8 P@g2 f1g2 h5f3 e2f3 f2f1b
info depth 21 seldepth 42 multipv 1 score cp 145 upperbound nodes 79258796 nps 639379 hashfull 999 tbhits 0 time 123962 pv d8f8 g7f8 P@g2 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 P@f5
info depth 21 seldepth 42 multipv 1 score cp 120 upperbound nodes 80555384 nps 639587 hashfull 999 tbhits 0 time 125949 pv d8f8 g7f8 P@g2 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 h7h5 g4h5 R@h7 B@h6 g5f3 N@c5 d7c8 P@d7 h7d7 Q@e8 Q@d8 e8e6 P@f7 e6c6 P@g6 h5g4 f3e5 g4h3 e5c6 B@a6 a8b8 c5b7 Q@f5 P@g4 f5g4
info depth 21 seldepth 46 multipv 1 score cp 86 upperbound nodes 82567914 nps 639694 hashfull 999 tbhits 0 time 129074 pv d8f8 g7f8 P@g2 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 h7h5 g4h5 R@h7 B@h6 P@g6
info depth 21 seldepth 47 multipv 1 score cp 42 upperbound nodes 88161838 nps 640148 hashfull 999 tbhits 0 time 137721 pv d8f8 g7f8 P@g2 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 c6e5 g4h5 g5f3 B@a4 P@c6 a4c6 b7c6 N@c5 d7e8 P@d7 e5d7 P@f7 e8f8 c5d7 f8f7 N@e5 f3e5 d7e5 f7f6
info depth 21 seldepth 47 multipv 1 score cp -15 upperbound nodes 94456993 nps 640117 hashfull 999 tbhits 0 time 147562 pv d8f8 g7f8 P@g2 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 h7h5
info depth 21 seldepth 47 multipv 1 score cp -90 upperbound nodes 103355498 nps 640777 hashfull 999 tbhits 0 time 161297 pv d8f8 g7f8 P@g2 f1g2 h5f3 e2f3 P@h3 g2h3 N@g5 h3g4 c6e5 g4h5 g5f3
info depth 21 seldepth 47 multipv 1 score cp -29 lowerbound nodes 112933675 nps 639916 hashfull 999 tbhits 0 time 176482 pv d8f8 g7f8 P@d6 e2e3
info depth 21 seldepth 47 multipv 1 score cp -185 upperbound nodes 145423519 nps 637151 hashfull 999 tbhits 0 time 228240 pv d8f8 g7f8 P@d6 Q@g7 R@f7 B@h3 d5d4 g3h4 f7g7 N@f6 d7d8
info depth 21 seldepth 47 multipv 1 score cp -100 nodes 156439578 nps 636466 hashfull 999 tbhits 0 time 245794 pv d8f8 g7f8 P@d6 Q@g7 R@f7 B@h3 f7g7 f8g7 N@d2 f1g2 Q@g6 e2e3 h5f3 g2f2 P@g4 N@f4 h4g5 f4g6 g5e3 f2e3 Q@g5 B@f4 g5g6 e3d2 g6g7 P@f5 f3h1 f5e6 d7e6
```

**Comment by ddugovic** [comment]
Fixed in 746797b63131cb4b04c8f73312e091fe6e7dd3ad (capture move selector no longer biases based upon distance from kings, which could have caused a pro-drop / anti-capture bias overall)
